### DISCUSSION TITLE
zoom-in/out timeline plugin with Custom container

### DISCUSSION THREAD

**Discussion by deepmedia-palances** [katspaugh/wavesurfer.js#3026] [Q&A]
Hi All,

I'm using the timeline plugin with a custom container(not the default built on the waveform). Want to know how to zoom in/out of the timeline when the original trigger zoom?

The timeline only zooms when it is built upon the original waveform. I would like to hear the community's solution to this.

Thanks

**Comment by katspaugh** [comment]
Timeline zooming and panning in a custom container isn't supported as both depend on the size and scroll position of the waveform container div.
You could subscribe to the zoom and scroll wavesurfer events to set the width and scroll your custom container accordingly.
